package.json

```diff
@@ -46,6 +46,7 @@
 	},
 	"dependencies": {
 		"@jedmao/location": "^3.0.0",
+		"@jest/globals": "^29.7.0",
```

Comment:
I am wondering if this locks this package into supporting Jest v29 until we update this to v30 when it comes around and so forth. Maybe this should be a peerDependency. This may need investgation.